src/dealDashboard/dealVotes/dealVotes.ts

```diff
@@ -17,73 +17,75 @@ export class DealVotes {
-      statusText: "Deal is approved",
-    },
-    {
-      condition: () => this.deal.isFailed,
-      voteText: "Voting is closed. You can find the voting history below.",
-      statusText: "Token Swap Failed",
-    },
-    {
-      condition: () => !this.deal.majorityHasVoted && !this.deal.isAuthenticatedRepresentativeUser && !this.deal.isAuthenticatedProposalLead,
-      voteText: "Waiting for the representatives to vote",
-      statusText: "Voting is progress",
-    },
-    {
-      condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedRepresentativeUser && this.deal.hasRepresentativeVoted && !this.deal.isAuthenticatedProposalLead,
-      voteText: "You have cast your vote. Please wait for other representatives to cast theirs. You are able to change your vote before the funding phase is initiated. Once the deal is approved, the Proposal Lead will initiate the funding phase",
-      statusText: "Voting in progress",
-    },
-    {
-      condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedRepresentativeUser && !this.deal.hasRepresentativeVoted && !this.deal.isAuthenticatedProposalLead,
-      voteText: "The deal will be approved once the majority has voted in favor. Once the deal is approved, the Proposal Lead will initiate the funding phase",
-      statusText: "Voting in progress",
-    },
-    {
-      condition: () => !this.deal.majorityHasVoted && this.deal.isAuthenticatedProposalLead,
-      voteText: "The deal will be approved once the majority has voted in favor. Once the deal is approved, you can initiate the funding phase",
-      statusText: "Voting in progress",
-    },
-    {
-      condition: () => this.deal.majorityHasVoted && this.deal.isAuthenticatedRepresentativeUser && !this.deal.isAuthenticatedProposalLead,
-      voteText: "Waiting for the Proposal Lead to initiate the funding phase.  You may still change your vote if you wish to",
-      statusText: "Voting is completed",
-    },
-    {
-      condition: () => this.deal.majorityHasVoted && !this.deal.isAuthenticatedRepresentativeUser && !this.deal.isAuthenticatedProposalLead,
-      voteText: "Waiting for the Proposal Lead to initiate the funding phase",
-      statusText: "Voting is completed",
-    },
-    {
-      condition: () => this.deal.majorityHasVoted && this.deal.isAuthenticatedRepresentativeUser && this.deal.isAuthenticatedProposalLead,
-      voteText: "You can now initiate the funding phase. You may still change your vote if you wish to",
-      statusText: "Voting is completed",
-    },
-    {
-      condition: () => this.deal.majorityHasVoted && !this.deal.isAuthenticatedRepresentativeUser && this.deal.isAuthenticatedProposalLead,
-      voteText: "You can now initiate the funding phase",
-      statusText: "Voting is completed",
-    },
-  ];
+  get everyTextCopy() {
+    return [
```

Comment:
FYI, changing this to getter made `get statusText` trigger on change

Reply:
where did yuo change it?

Reply:
`everyTextCopy` was just a normal class variable before